pallets/pswap-distribution/src/lib.rs

```diff
@@ -478,7 +478,7 @@ pub mod pallet {
         const PSWAP_BURN_PERCENT: Percent;
         type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
         type GetIncentiveAssetId: Get<AssetIdOf<Self>>;
-        type GetTBCDAssetId: Get<AssetIdOf<Self>>;
+        type GetVXORAssetId: Get<AssetIdOf<Self>>;
```

Comment:
Maybe set some universal name? Like `GetBuyBackAssetId`?
Because we cannot be sure that this asset will be not changed in the future, it doesn't make sense to change names every time.